platforms/hyperledger-fabric/configuration/add-new-channel.yaml

```diff
@@ -0,0 +1,112 @@
+# This playbook is a subsequent playbook
+##########################################################################################
+# DO NOT RUN THIS IF YOU HAVE NOT RUN deploy-network.yaml and deployed the Fabric network
+# and want to add a new channel to the existing network
+##########################################################################################
+# This playbook can only be run after all pods and orderers are available
+# Please use the same network.yaml to run this playbook as used for deploy-network.yaml
```

Comment:
Dont think we will use the "same" network.yaml, surely new channels should be there.

Reply:
Fixed it.